### ISSUE TITLE
[Question] [Enhancement] Aerodynamics

### ISSUE THREAD

**Issue by AlessandroZavoli** [google/brax#163]
Is it possible to add custom aerodynamics to an object, e.g., to model wing-based objects like airplanes or helicopters?

**Comment by soldierofhell** [comment]
Do you mean something like [lift&drug plugin](https://gazebosim.org/tutorials?tut=aerodynamics&cat=physics) in gazebo?

**Comment by AlessandroZavoli** [comment]
Yes, something like that. I could provide the mathematical equations for a few cases of interest (e.g., flat plate with no lift, 2D airfoil) or a generic model in terms of classical aerodynamic coefficients.

This is probably lower in priority than [issues#156](https://github.com/google/brax/issues/156), that is, fixing second-order terms in the Euler equations for attitude dynamics.
